MIGRATION.md

```diff
@@ -1 +1,11 @@
-# Миграция
+# Migration
+
+## 2.1.0
+
+The `width` modifier could be used independent of the theme `islands` in the `input`, `select` and `textarea` blocks. To use this modifier dynamically, add it to the dependencies:
```

Comment:
To use this modifier dynamically, add it to the dependencies: —> Add this modifier to the dependencies to generate BEMJSON dynamically: